### PR TITLE
feat(uat): add tags section to uat readme

### PR DESCRIPTION
**Pull request by mzyl13**
**Issue #, if available:**
https://klika-tech.atlassian.net/browse/GGMQ-278
Update README.md file

**Description of changes:**
- Update README.md of uat by adding "Tags" section

**Why is this change necessary:**
We can use tags to run exact scenarios

**How was this change tested:**
Documentation update, will review by Team

**Any additional information or context required to review the change:**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by auarbekov — DISMISSED** [review]

**Review by eschastlivtsev — DISMISSED** [review]

File: uat/README.md [hunk 1]
```diff
@@ -117,18 +117,52 @@ On Windows due to mosquitto-based client is not yet build on Windows use a littl
...
-Dtest.log.path - path where you would like the test results to be stored.  
+### Command arguments:
+Dggc.archive - path to the nucleus zip that was downloaded.
+Dtest.log.path - path where you would like the test results to be stored.
 
+### Tags
+-Dtags can be extended if you would like to test exact scenarios or exclude some scenarios:
 
--Dtags can be extended, if you would like to test exact scenario, you can do as follows:
+GGMQ - to start all test scenarios run
```
Comment (bgklika) [inline uat/README.md]: at GGMQ

File: uat/README.md [hunk 2]
```diff
@@ -117,18 +117,52 @@ On Windows due to mosquitto-based client is not yet build on Windows use a littl
...
+Dtest.log.path - path where you would like the test results to be stored.
 
+### Tags
+-Dtags can be extended if you would like to test exact scenarios or exclude some scenarios:
 
--Dtags can be extended, if you would like to test exact scenario, you can do as follows:
+GGMQ - to start all test scenarios run 
+Example:
+```bash
+sudo -E java -Dggc.archive=greengrass-nucleus-latest.zip -Dtest.log.path=logs -Dtags=GGMQ -jar testing-features/target/client-devices-auth-testing-features.jar
```
Comment (bgklika) [inline uat/README.md]: -Dtags=@GGMQ

File: uat/README.md [hunk 3]
```diff
@@ -117,18 +117,52 @@ On Windows due to mosquitto-based client is not yet build on Windows use a littl
...
+```
+
+To choose running of specific tests using the following tags:
+
+@sdk-java, @mosquitto-c, @paho-java, @paho-python - to choose client to run test scenarios
+Example:
+```bash
+sudo -E java -Dggc.archive=greengrass-nucleus-latest.zip -Dtest.log.path=logs -Dtags="GGMQ and @sdk-java" -jar testing-features/target/client-devices-auth-testing-features.jar
```
Comment (bgklika) [inline uat/README.md]: -Dtags=@GGMQ...

File: uat/README.md [hunk 4]
```diff
@@ -117,18 +117,52 @@ On Windows due to mosquitto-based client is not yet build on Windows use a littl
...
+```
+
+@mqtt3, @mqtt5 - to choose protocol version for test scenarios
+Example:
+```bash
+sudo -E java -Dggc.archive=greengrass-nucleus-latest.zip -Dtest.log.path=logs -Dtags="GGMQ and @mqtt5" -jar testing-features/target/client-devices-auth-testing-features.jar
```
Comment (bgklika) [inline uat/README.md]: -Dtags=@GGMQ...

File: uat/README.md [hunk 5]
```diff
@@ -117,18 +117,52 @@ On Windows due to mosquitto-based client is not yet build on Windows use a littl
...
+```
+
+@GGMQ-1-T(scenario number) - to specify scenario number that should be run
+Example: 
+```bash
+sudo -E java -Dggc.archive=greengrass-nucleus-latest.zip -Dtest.log.path=logs -Dtags="GGMQ and @GGMQ-1-T1" -jar testing-features/target/client-devices-auth-testing-features.jar
```
Comment (bgklika) [inline uat/README.md]: GGMQ not required in tags here, single @GGMQ-1-T1 is enough

File: uat/README.md [hunk 6]
```diff
@@ -117,18 +117,52 @@ On Windows due to mosquitto-based client is not yet build on Windows use a littl
...
+```
 
+@SkipOnWindows - to skip clients that are not supported by Windows
```
Comment (bgklika) [inline uat/README.md]: clients or tests

File: uat/README.md (hunk 2 above)
Comment (mzyl13) [reply uat/README.md]: fixed

File: uat/README.md (hunk 1 above)
Comment (mzyl13) [reply uat/README.md]: fixed

File: uat/README.md (hunk 3 above)
Comment (mzyl13) [reply uat/README.md]: fixed

File: uat/README.md (hunk 4 above)
Comment (mzyl13) [reply uat/README.md]: fixed

File: uat/README.md (hunk 5 above)
Comment (mzyl13) [reply uat/README.md]: fixed

File: uat/README.md (hunk 6 above)
Comment (mzyl13) [reply uat/README.md]: fixed

**Review by bgklika — APPROVED** [review]

**Comment by bgklika** [comment]
@OffTheNetwork is missing. That tag mark scenarios which block and unblock non-local TCP connections over port 8883.

**Review by auarbekov — APPROVED** [review]

**Review by eschastlivtsev — APPROVED** [review]

**Pull request closed without merging**